src/core/createInstance.js

```diff
@@ -29,7 +29,8 @@ export default (
   logger,
   window
 ) => {
-  let errorsEnabled;
+  // Assume errors are enabled until configuration says otherwise.
+  let errorsEnabled = true;
```

Comment:
Previously, it was `undefined` until the configure command executed, so the logic [here](https://github.com/adobe/alloy/blob/1e24d20e2e0c97496bf3b6a9489b537cbbc17d7c/src/core/createInstance.js#L177-L181) decided not to reject the promise.